synology-letsencrypt.sh

```diff
@@ -18,26 +22,26 @@ if [[ ! -s $cert_id_path ]]; then
 fi
 source "$cert_id_path"
 
-
-## install hook
+## install hook if necessary
 archive_path="/usr/syno/etc/certificate/_archive/$cert_id"
 if [[ ! -d $archive_path ]]; then
     mkdir -p "$archive_path"
 fi
 
-cat > "$hook_path" <<EOF
-#!/bin/bash
+if [[ ! -x $hook_path ]]; then
```

Comment:
I assume that you want to skip overwriting the hook because you are producing certs for another machine, and have changed it to a no-op?

We want to be able to update/maintain the hook, it is an implementation detail that might change with a lego or synology future change. 

I think this is a prime candidate for a second argument to the script, where we simply skip adding the hook to the lego command, i.e. a "do not install certificate on this machine" getopt argument.

Reply:
I would actually all things considered probably move the hook script creation into install.sh; I've applied your suggestion here, but it makes more sense to have it be in the install, imo.